website/.env.test

```
@@ -0,0 +1,24 @@
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN="demo-social-income-local"
+NEXT_PUBLIC_FIREBASE_PROJECT_ID="demo-social-income-local"
+NEXT_PUBLIC_FIREBASE_AUTH_EMULATOR_URL="http://localhost:9099"
+NEXT_PUBLIC_FIREBASE_FIRESTORE_EMULATOR_HOST="localhost"
+NEXT_PUBLIC_FIREBASE_FIRESTORE_EMULATOR_PORT="8080"
+NEXT_PUBLIC_FIREBASE_FUNCTIONS_EMULATOR_HOST="localhost"
+NEXT_PUBLIC_FIREBASE_FUNCTIONS_EMULATOR_PORT="5001"
+NEXT_PUBLIC_FIREBASE_STORAGE_EMULATOR_HOST="localhost"
+NEXT_PUBLIC_FIREBASE_STORAGE_EMULATOR_PORT="9199"
+NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET="demo-social-income-local.appspot.com"
+BASE_URL="http://localhost:3000"
+SENDGRID_LIST_ID="2896ee4d-d1e0-4a4a-8565-7e592c377e36"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)